### DISCUSSION TITLE
How to pick primaryColor from the first color of pre-defined colors?

### DISCUSSION THREAD

**Discussion by weeliem** [mantinedev/mantine#1221] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2240115/162627048-f8b8831a-392a-4e6e-9fea-379d2a627dfd.png)
Hi, may i know how can i pick the first brand color code for the primaryColor?

**Comment by rtivital** [comment, marked as answer]
```tsx
const theme = useMantineTheme();
theme.colors.brand[0];
```

https://mantine.dev/theming/mantine-provider/#use-mantine-theme-hook

**Reply by weeliem** [reply]
I tried, but encountered this error message.
![image](https://user-images.githubusercontent.com/2240115/162627694-7c63d2fd-ec93-4bbc-9ae6-44db3846f313.png)

The mantine theme hook is allowed to use in mantine provider?

**Reply by rtivital** [reply]
useMantineTheme can only be called inside components, like any other hook

**Reply by weeliem** [reply]
i see, okay got it. Thanks
